plugins/modules/rhel_facts.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: Red Hat Inc.
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: rhel_facts
```

Comment:
I am not too sure if we want `_facts` in the name since we got rid of it in favor of `_info`. What do you think?

Reply:
I went with `_facts` because it's not an info module that returns data to be registered via a task. Instead this should be set with the `ansible_facts_modules` variable at the play or inventory level similar to this example: https://github.com/jborean93/ansible-pkg_mgr-override/blob/main/method1.yml

Reply:
+1